ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -111,7 +117,15 @@ export class Send extends IronfishCommand {
     }
 
     if (flags.fee) {
-      fee = CurrencyUtils.decodeIron(flags.fee)
+      if (CurrencyUtils.decodeIron(flags.fee) < 1n) {
+        this.error(`The minimum fee is ${CurrencyUtils.renderOre(1n, true)}`)
+      }
+
+      fee = CurrencyUtils.encode(CurrencyUtils.decodeIron(flags.fee))
+    }
+
+    if (flags.feeRate) {
+      feeRate = CurrencyUtils.encode(CurrencyUtils.decodeIron(flags.feeRate))
```

Comment:
I dont think it simplify any codes. I will keep it here since this is validation check, all the validations are performed at beginning. We want to fail fast for the user.

Reply:
It's unclear you're using a side effect of the decoder to use validation. That type of engineering is going to lead another developer to delete this code.

I think when writing this code, it should lead you to realize that Oclif has a strategy for this, and they expect you to write a custom flag which we haven't taken advantage of yet. https://oclif.io/docs/flags#custom-flags

If we wrote a FeeRate flag, it would ensure every time we take in a fee rate it would be consistent, it would also be a good separation of input validation.